src/service.rs

```diff
@@ -202,6 +202,7 @@ impl Service for Slskd {
 #[derive(Debug, Clone)]
 pub struct Transmission {
     pub rpc_credentials: Option<Credentials>,
+    pub address: Option<String>,
```

Comment:
Maybe same here, an `IP` (with an optional port) `struct`/`enum` would be best, but it's not the end of the world if it's just a `String` (the `transmission-remote` call will yell at us then.